### ISSUE TITLE
Mesh goes black when using Tweak

### ISSUE THREAD

**Issue by PierreYvesMORA** [CGCookie/retopoflow#360]
Hello,

When I use Tweak the simplified mesh goes black and i cannot see vertices i am tweaking.

![capture d ecran 2017-09-03 a 18 19 55](https://user-images.githubusercontent.com/31594170/30004706-9f8a6bda-90d4-11e7-9036-ce20baaaedb3.png)

**Comment by vxlcoder** [comment]
Thank you for the report, @PierreYvesMORA.

To diagnose what is going on, please provide us a bit more information.  For example:

- Does this happen with any of the other tools?
- Does it turn black right away, or after you use the tool?
- If you run Blender from the terminal, are there any messages printed to the console?
- What are your system specs?
- Which version of Blender are you using?

**Comment by PierreYvesMORA** [comment]
Hi vxlcoder, thanks for your reply, here are the informations you asked for :
° It happen with Tweak, PolyPen and Polystrips, this problem is on the generated mesh. And on Polystrips and Polypen, if I do it in Object mode, the display is OK, but when doing it in Edit mode it's KO (and tweak is not available in object mode...).

° It turns black the moment I click the button Tweak (right away)

° When launching Blender from Terminal the messages are : 
_______________________________________
ndof: 3Dx driver not found
Read new prefs: /Users/pierre-yvesmora/Library/Application Support/Blender/2.76/config/userpref.blend
found bundled python: /Applications/blender.app/Contents/MacOS/../Resources/2.76/python
RetopoFlow Shader Info
  attribs: dotoffset, offset
  uniforms: object_scale, clip_start, perspective, clip_end
RetopoFlow updater verbose is enabled
Read in json settings from file
Running background check for update
_______________________________________

° I am running blender from an iMac (21.5 inches, late 2013), 16 Go RAM 1600 MHz DDR3, NVIDIA GeForce GT 750M 1024 Mo.

° The Blender version i am using is 2.76 but I tried with 2.78 too (same issue)

**Comment by PierreYvesMORA** [comment]
And Mac OS X Yosemite version 10.10.5

**Comment by vxlcoder** [comment]
Thanks for the data @PierreYvesMORA!

At this point, I'm confused about this bug.  Everything _seems_ OK (adequate hardware, recent Blender and OS, nothing unusual printed to console).  It works OK in object mode, but not in edit mode.  I haven't seen this particular bug before.

Are you able to make edits to the target, even though you cannot see individual faces, edges, and verts?

**Comment by vxlcoder** [comment]
Also, is this a recent bug, or has this been ongoing?

**Comment by PierreYvesMORA** [comment]
Yes I can make edits to the target, and I just started using retopoflow so it happened since I started using it. By the way it's an amazing tool, intuitive and very useful.

**Comment by vxlcoder** [comment]
I believe this issue applies to v1.x only, so I'm going to close this issue now that v2.0 is officially released (see `release/2.0.0` branch or Blender Market page).  If this is still an issue with v2.0, please reopen, and post more details.  Thanks for reporting this issue, and we look forward to seeing how v2.0 helps your retopo workflow!

**Comment by PierreYvesMORA** [comment]
OK thanks a lot ! I will test this release then.
